crates/turborepo-repository/src/package_graph/builder.rs

```diff
@@ -146,22 +180,19 @@ impl<'a, S> BuildState<'a, S> {
     }
 }
 
-impl<'a> BuildState<'a, ResolvedPackageManager> {
+impl<'a, T: PackageDiscovery> BuildState<'a, ResolvedPackageManager, T> {
```

Comment:
I see a future where the `ResolvedPackageManager` state is no more and we go straight to the `ResolvedWorkspaces`. Not yet, but this lays the groundwork for that simplification.